src/main/java/edu/tseidler/states/Running.java

```diff
@@ -12,6 +12,10 @@ public Running(GameState previousState) {
 
     @Override
     GameState getNextState() {
+        board = new Board(new int[] {5, 15, 3});
```

Comment:
to new Board pewnie jest tylko tymczasowe, aby sprawdzić działanie, ale podejrzewam, że docelowo jakieś wstrzyknięcie będzie tu na miejscu :)

Reply:
tak, chciałem tylko sprawdzić czy rysowanie robi robotę dobrze ;)